doc/user-guide/dask.rst

```diff
@@ -2,195 +2,151 @@
+--------
+
+- `Zonal averaging with the NOAA National Water Model <https://docs.coiled.io/user_guide/xarray.html?utm_source=xarray-docs>`_
+- `CMIP6 Precipitation Frequency Analysis <https://gallery.pangeo.io/repos/pangeo-gallery/cmip6/precip_frequency_change.html>`_
+- `Using Dask + Cloud Optimized GeoTIFFs <https://gallery.pangeo.io/repos/pangeo-data/landsat-8-tutorial-gallery/landsat8.html#Dask-Chunks-and-Cloud-Optimized-Geotiffs>`_
+- `Xarray + Dask Tutorial <https://tutorial.xarray.dev/intermediate/xarray_and_dask.html>`_
+
+
+Using Dask with Xarray
+----------------------
 
-.. image:: ../_static/dask_array.png
-   :width: 40 %
+.. image:: https://docs.dask.org/en/stable/_images/dask-array.svg
+   :width: 50 %
    :align: right
    :alt: A Dask array
 
-Dask divides arrays into many small pieces, called *chunks*, each of which is
-presumed to be small enough to fit into memory.
+Xarray operations on Dask-backed arrays are lazy. This means computations are not executed immediately, but are instead queued up as tasks in a Dask graph.
 
-Unlike NumPy, which has eager evaluation, operations on Dask arrays are lazy.
-Operations queue up a series of tasks mapped over blocks, and no computation is
-performed until you actually ask values to be computed (e.g., to print results
-to your screen or write to disk). At that point, data is loaded into memory
-and computation proceeds in a streaming fashion, block-by-block.
+When a result is requested (e.g., for plotting, saving, or explicitly computing), Dask executes the task graph. The computations are carried out in parallel, with each chunk being processed independently. This parallel execution is key to handling large datasets efficiently.
 
-The actual computation is controlled by a multi-processing or thread pool,
-which allows Dask to take full advantage of multiple processors available on
-most modern computers.
-
-For more details, read the `Dask documentation <https://docs.dask.org/>`__.
-Note that xarray only makes use of ``dask.array`` and ``dask.delayed``.
+Nearly all Xarray methods have been extended to work automatically with Dask Arrays. This includes things like indexing, concatenating, rechunking, grouped operations, etc. Common operations are covered in more detail in each of the sections below.
 
 .. _dask.io:
 
 Reading and writing data
-------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-The usual way to create a ``Dataset`` filled with Dask arrays is to load the
-data from a netCDF file or files. You can do this by supplying a ``chunks``
-argument to :py:func:`~xarray.open_dataset` or using the
-:py:func:`~xarray.open_mfdataset` function.
+When reading data, Dask automatically divides your dataset into smaller chunks. These chunks are essentially small, manageable blocks of the larger dataset. You can specify the size of chunks with the ``chunks`` argument (see the `Dask Array docs on chunks <https://docs.dask.org/en/stable/array-chunks.html?utm_source=xarray-docs>`_).
```

Comment:
Whilst this is a good explanation, reading data is not the only way to generate dask chunks, so I'm not sure the first explanation of what a chunk is should go here.